Cargo.toml

```diff
@@ -37,13 +40,25 @@ panic-probe = { version = "0.3.1", features = ["print-defmt"] }
 tinybmp = "0.5.0"
 # Used by the noise_i2c examples
 rand = { version = "0.8.4", default-features = false, features = [ "small_rng" ] }
+embassy-embedded-hal = { version = "0.2.0", git = "https://github.com/embassy-rs/embassy" }
```

Comment:
Why git dependency?

Reply:
Because 0.2 wasn't even on crates.io when most of this was done ;)

I could get rid of the git dependency of embedded-hal (by disabling the default features); embassy-executor still has to come from git as it has to agree with embassy-stm32 for the time driver

Reply:
Ah, indeed. I have only tried with embassy-nrf and esp-hal.